src/docs/src/documentation/content/xdocs/zookeeperAdmin.xml

```diff
@@ -1076,6 +1076,26 @@ server.3=zoo3:2888:3888</programlisting>
             </listitem>
           </varlistentry>
 
+          <varlistentry>
+            <term>zookeeper.ipReachableTimeout</term>
```

Comment:
Thanks @bensherman for pointing this out. Fixed in commit 677b5ccb7ce214eee750fda70eba3643d8c7a6c0.